### PR TITLE
Incorrect query param

### PR DESCRIPTION
**Pull request by bbalakriz**
Hi, The query parameter for the second line approval request has "john=admin", however, as per the constructs, it should be "user=john". Please review the proposed change. Thanks!

### CODE REVIEW

File: kogito-usertasks-quarkus/README.md [hunk 1]
```diff
@@ -142,14 +142,14 @@ This should return empty response as the admin user was the first approver and b
 Repeating the request with another user will return task
 
 ```
-curl -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/approvals/{uuid}/tasks?john=admin&group=managers'
+curl -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/approvals/{uuid}/tasks?user=john=admin&group=managers'
```
Comment (cristianonicolai) [inline kogito-usertasks-quarkus/README.md]:
```suggestion
curl -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/approvals/{uuid}/tasks?user=john&group=managers'
```

**Review by cristianonicolai — COMMENTED** [review]
@bbalakriz just one minor tweak, thanks!

**Comment by radtriste** [comment]
This should be solved here https://github.com/kiegroup/kogito-examples/pull/175 :)

**Comment by cristianonicolai** [comment]
@radtriste sure, it van be rebased :)

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**